cli/initiator/reshare.go

```diff
@@ -0,0 +1,231 @@
+package initiator
+
+import (
+	"crypto/rsa"
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+
+	"github.com/bloxapp/ssv-dkg/cli/flags"
+	cli_utils "github.com/bloxapp/ssv-dkg/cli/utils"
+	"github.com/bloxapp/ssv-dkg/pkgs/crypto"
+	"github.com/bloxapp/ssv-dkg/pkgs/initiator"
+	"github.com/bloxapp/ssv-dkg/pkgs/utils"
+)
+
+func init() {
+	flags.InitiatorPrivateKeyFlag(StartReshare)
```

Comment:
for example all this code is very repeatable, all this functions are the same just with different const, maybe we should also just pass a param in instead of having a function for each?